JP1033C, JP0847B

KonstantinIvanchenko opened the pull request.

Commit:
JP1033C, JP0847B

KonstantinIvanchenko committed:
added links

KonstantinIvanchenko committed:
fix end of line

KonstantinIvanchenko committed:
fix end of line

vvssttkk committed:
Update cd-stage.yml

vvssttkk commented on mkdocs.yml:
```diff
@@ -206,3 +207,4 @@ nav:
           - python/special/import/JP0837B-с_уязвимостью_httpoxy.md
           - python/special/import/JP0839B-crypto.md
           - python/special/import/JP0843B-pyghmi.md
+          - python/special/import/JP0847B-import_all.md
```
мб лучше всунуть около JP1231C (это получается «Класс «Вызов Функции»»)?

vvssttkk committed:
коррекция JP1033C

dependabot[bot] committed:
dependabot: bump mkdocs-material from 9.5.9 to 9.5.10 (#33)

Bumps [mkdocs-material](https://github.com/squidfunk/mkdocs-material) from 9.5.9 to 9.5.10.
- [Release notes](https://github.com/squidfunk/mkdocs-material/releases)
- [Changelog](https://github.com/squidfunk/mkdocs-material/blob/master/CHANGELOG)
- [Commits](https://github.com/squidfunk/mkdocs-material/compare/9.5.9...9.5.10)

---
updated-dependencies:
- dependency-name: mkdocs-material
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: vvssttkk <8581044+vvssttkk@users.noreply.github.com>

vvssttkk commented on docs/python/common_errors/calls/libs/JP1033C-json_load_large.md:
```diff
@@ -0,0 +1,25 @@
+![Static Badge](https://img.shields.io/badge/%D0%94%D0%BE%D1%81%D1%82%D0%BE%D0%B2%D0%B5%D1%80%D0%BD%D0%BE%D1%81%D1%82%D1%8C%20%D0%BE%D0%BF%D1%80%D0%B5%D0%B4%D0%B5%D0%BB%D0%B5%D0%BD%D0%B8%D1%8F-%D0%BD%D0%B8%D0%B7%D0%BA%D0%B0%D1%8F-mediumblue?style=for-the-badge)
+![Static Badge](https://img.shields.io/badge/%D0%94%D0%BE%D1%81%D1%82%D0%BE%D0%B2%D0%B5%D1%80%D0%BD%D0%BE%D1%81%D1%82%D1%8C%20%D0%BE%D0%BF%D1%80%D0%B5%D0%B4%D0%B5%D0%BB%D0%B5%D0%BD%D0%B8%D1%8F-%D1%81%D1%80%D0%B5%D0%B4%D0%BD%D1%8F%D1%8F-orange?style=for-the-badge)
+
+Проверка использования `json.load`. Злоумышленник может воспользоваться данной конструкцией для загрузки файла большого размера, что может повлечь к чрезмерному использованию системных ресурсов.
```
```suggestion
Проверка использования `json.load`. Злоумышленник может воспользоваться данной конструкцией для загрузки файла большого размера, что может привести к чрезмерному использованию системных ресурсов.
```

vvssttkk committed:
Apply suggestions from code review
